website/content/docs/discovery/dns.mdx

```diff
@@ -394,6 +497,9 @@ you can use the following query formats specify namespace but not partition:
   [<tag>.]<service>.service.<namespace>.<domain>
   ```
 
+To lookup a namespaced service in a cluster peer,
+the [canonical format](#canonical-format) must be used instead.
```

Comment:
```suggestion
Use the [canonical format](#canonical-format) to lookup services bound to a namespace in a cluster peer.
```
It's a bit of a gray area, but I think "namespace" is a noun. I recall seeing it in the K8s docs as verb, but it read as poor grammar to me. "Bound" might not be the right verb, either. I won't die on this hill if you disagree.